### PR TITLE
Add new layout for base item adapter

### PR DESCRIPTION
**Pull request by Dollym89**

### CODE REVIEW

File: app/src/main/java/com/guru/cocktails/ui/base/BaseListAdapter.kt [hunk 1]
```diff
@@ -24,7 +24,7 @@ abstract class BaseListAdapter<ListItem : BaseListItem>(
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int) =
         ViewHolder(parent.inflate(getListItemLayout()))
 
-    open fun getListItemLayout() = R.layout.item_coctail_grid
+    open fun getListItemLayout() = R.layout.base__thumb_item
```
Comment (Petrulak) [inline app/src/main/java/com/guru/cocktails/ui/base/BaseListAdapter.kt]: lets use naming conventions from here https://jeroenmols.com/blog/2016/03/07/resourcenaming/

File: app/src/main/res/layout/base__thumb_item.xml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:card_view="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/container"
+    android:layout_width="wrap_content"
```
Comment (Petrulak) [inline app/src/main/res/layout/base__thumb_item.xml]: we also have styles for that defined in styles. in this case "@style/WrapContent"

File: app/src/main/res/values/dimen.xml [hunk 3]
```diff
@@ -1,71 +1,78 @@
...
-  <dimen name="avator_size">40dp</dimen>
-  <dimen name="card_title_height">40dp</dimen>
-  <dimen name="article_titles">20sp</dimen>
-  <dimen name="cards_button_width">48dp</dimen>
-  <dimen name="cards_button_height">48dp</dimen>
-  <dimen name="article_subheading">16sp</dimen>
-  <dimen name="card_image_height">200dp</dimen>
-  <dimen name="list_body">14sp</dimen>
-  <dimen name="navheader_height">222dp</dimen>
+    <dimen name="activity_horizontal_margin">16dp</dimen>
```
Comment (Petrulak) [inline app/src/main/res/values/dimen.xml]: we can also have the same style for XML,  you can set space, indent and continues indent to `2`.

Comment (Dollym89) [reply app/src/main/res/values/dimen.xml]: I dont see point at setting specific dimension for layout when we can use global prespecified dimen

**Pull request merged**